images/lutris/Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG BASE_APP_IMAGE
+
+# hadolint ignore=DL3006
+FROM ${BASE_APP_IMAGE}
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Get the latest wine && winetricks
+# The following is based on gloriouseggroll's advice on how to get out of wine dependency hell
+RUN dpkg --add-architecture i386
```

Comment:
It may be easier to maintain this going forward if we move it to a style more like the other containers (a single `RUN` command, a `REQUIRED_PACKAGES` arg, standardized setup/cleanup commands).

Reply:
Got it. I can see how you're styling this - so will follow this from now on. I have another branch I'm working on too for some other functionality I'll PR with you guys at a later date.